src/lib/data.ts

```diff
@@ -522,6 +522,7 @@ termination_socket.on('remote_storage/update', (row:RemoteKV) => {
     safeLocalStorageSet(`ogs-remote-storage-store.${user.id}.${row.key}`, JSON.stringify(row));
 
     if (last_modified < row.modified) {
+        safeLocalStorageSet(`ogs.${row.key}`, row.value);
```

Comment:
This needs to be moved up to the above if statement so we only store the value if `if (row.replication === Replication.REMOTE_OVERWRITES_LOCAL) `

Reply:
Also that needs to be json stringified I think

Reply:
🤦‍♀️  Of course, done.

Reply:
Oh, I missed the stringify comment, I'll take a look